doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -42,6 +42,7 @@ Board support
 
   * Bias-pull-up for Thingy:91 X nRF9151 UART RX pins.
   * Alternative partition tables for Thingy:91 X.
+  * Runtime BLE/WiFi switch power management for Thingy:91 X.
```

Comment:
```suggestion
  * Runtime Bluetooth LE/Wi-Fi switch power management for Thingy:91 X.
```

Reply:
Seems like "Wi-Fi" term is not updated correctly.

Reply:
Missed that one in a rebase, thanks!